src/main/scala/scalikejdbc/athena/Config.scala

```diff
@@ -81,17 +82,24 @@ object Config {
   val ComplexTypeColumnLength = "ComplexTypeColumnLength"
   val ConnectionTest = "ConnectionTest"
   val ConnectTimeout = "ConnectTimeout"
+  val IdPHost = "IdP_Host"
+  val IdPPort = "IdP_Port"
   val LogLevel = "LogLevel"
   val LogPath = "LogPath"
   val MaxCatalogNameLength = "MaxCatalogNameLength"
   val MaxColumnNameLength = "MaxColumnNameLength"
   val MaxErrorRetry = "MaxErrorRetry"
+  val MaxQueryExecutionPollingInterval = "MaxQueryExecutionPollingInterval"
   val MaxSchemaNameLength = "MaxSchemaNameLength"
+  val MaxStreamErrorRetry = "MaxStreamErrorRetry"
   val MaxTableNameLength = "MaxTableNameLength"
   val MetadataRetrievalMethod = "MetadataRetrievalMethod"
   val NonProxyHosts = "NonProxyHosts"
+  val Password = "Password"
   val PWD = "PWD"
```

Comment:
`PWD` and `UID` seems to have been removed...? 🤔

Oh, I found [the following description](https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC_2.0.7/docs/Simba+Athena+JDBC+Driver+Install+and+Configuration+Guide.pdf).

* Password

> As an alternative, you can configure this property using the alias `PWD`.

* User

> As an alternative, you can configure this property using the alias `UID`.

It is desirable to leave these. 👍